Tests/MongoSwiftSyncTests/SpecTestRunner/SpecTest.swift

```diff
@@ -1,3 +1,4 @@
+// swiftlint:disable duplicate_imports
```

Comment:
I think this is because it doesn't distinguish between these two lines, even though they have different meanings:
```
import MongoSwiftSync
@testable import struct MongoSwiftSync.MongoClientOptions
```